src/widgetastic_patternfly/__init__.py

```diff
@@ -122,7 +122,8 @@ def active(self):
 
     @property
     def disabled(self):
-        return self.browser.get_attribute('disabled', self) == 'disabled'
+        return 'disabled' in self.browser.classes(self) or (self.browser.get_attribute
+                                                            ('disabled', self) == 'disabled')
```

Comment:
the indentation here iis ugly, please, end the line with or and put the second condition on its line as whole :)

Reply:
Done @mfalesni